app/shop.py

```diff
@@ -0,0 +1,46 @@
+from __future__ import annotations
+import datetime
+
+import app.customer as cust
+
+
+class Shop:
+    shops = {}
+
+    def __init__(
+            self,
+            name: str,
+            location: list[int],
+            products: dict
+    ) -> None:
+        self.__name = name
+        self.__location = location
+        self.__products = products
+        Shop.shops[self.__name] = self
+
+    @property
+    def products_price(self) -> dict:
+        return self.__products

```

Comment:
okay, if you want to use getters, I propose adding setters for the validation of incorrect data. WDYT?)